browser/model/devstudio.js

```diff
@@ -39,6 +38,10 @@ class DevstudioInstall extends InstallableItem {
         return this.postJDKInstall(installer, result);
       })
       .then(() => {
+        let runtimeproperties = path.join(this.installerDataSvc.devstudioDir(), 'studio', 'runtime_locations.properties');
```

Comment:
Please remove all the new code. It should be done only once in jbosseap.js installer.